packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/helpers.ts

```diff
@@ -225,3 +229,30 @@ function formatTaxToMinorUnits(tax: number): number {
 function formatTaxFromMinorUnits(tax: number): number {
   return tax / 100;
 }
+
+export const isFieldRequired = <
+  TFieldNames extends string,
+  TOcrRequiredFields extends Record<TFieldNames, boolean>,
+  TFieldValues extends FieldValues
+>(
+  fieldName: TFieldNames,
+  ocrRequiredFields: TOcrRequiredFields | undefined,
+  value?: FieldValue<TFieldValues>
+) => {
+  if (value) return false;
+
+  return (
+    ['invoiceNumber', 'dueDate', 'tags', 'currency'].includes(fieldName) ||
```

Comment:
By default, we have no predefined fields.

Reply:
We have fields that are marked as required before, they were predefined

Reply:
We have fields that are marked as required before, they were predefined. Can you please navigate, what we should do in this case?

Reply:
The idea is to keep backward compatibility as full as possible. This new functionality could be easily reached with the options in the theme. So, by default, we should not display these fields as required for existing customers until they configured them.

Reply:
I’ve added the change you suggested. I noticed that we previously had default required fields. Could you help me understand why we need to change them?

Also this is coming from your sugestion:
<img width="824" alt="Screenshot 2024-08-23 at 21 53 22" src="https://github.com/user-attachments/assets/3cac9699-7367-491c-bc9c-7ca4993a39b8">